Makefile

```diff
@@ -1,9 +1,12 @@
+.POSIX:
```

Comment:
.POSIX sets the CC to c99. It is not available everywhere, for example, clang does not have such an alias.